.github/workflows/deploy.yml

```diff
@@ -47,11 +47,18 @@ jobs:
       run: |
         curl "https://labs.w3.org/spec-generator/?type=respec&url=https://raw.githack.com/${{ github.repository }}/${{ github.sha }}/aria-practices.html" -o deploy/index.html -f --retry 3
 
+
+    - name: Run PostCSS
+      run: |
+        cd deploy
```

Comment:
Could also move this section right into the next block too, if the double `cd` might cause an issue with the runner logic